.env.example

```
@@ -0,0 +1,223 @@
+DAYTONA_AUTO_STOP_INTERVAL_MINUTES=
+DAYTONA_AUTO_ARCHIVE_INTERVAL_MINUTES=
+
+# Vercel Sandboxes. Cloudflare: secret var.vercel_sandbox_token;
+# var.vercel_sandbox_project_id, var.vercel_sandbox_team_id,
+# var.vercel_base_snapshot_id, var.vercel_sandbox_runtime,
+# var.vercel_sandbox_api_base_url, var.vercel_snapshot_expiration_ms
+VERCEL_TOKEN=
+VERCEL_PROJECT_ID=
+VERCEL_TEAM_ID=
+VERCEL_BASE_SNAPSHOT_ID=
+VERCEL_BASE_SNAPSHOT_NAME=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VERCEL_BASE_SNAPSHOT_NAME key should go before the VERCEL_PROJECT_ID key

(UnorderedKey)